google/ggrc-core
# GGRC-6648 Remove initial login screen for authenticated users

# Issue description

Remove initial login screen for authenticated users and redirect them to dashboard.
For not authenticated users - show login page, authenticate, redirect to dashboard.

# Steps to test the changes

Before changes: log in the app, then open http://localhost:8080/. The login screen should appear.
After changes: log in the app, then open http://localhost:8080/. You should be redirected to dashboard without seeing login screen again.

# Solution description

Add check to find out whether user is already logged in. If yes, and user is authenticated in the system, redirect to dashboard.

# Sanity checklist




# PR Review checklist
